### DISCUSSION TITLE
登录提示 HighRiskEnvironment怎么解决

### DISCUSSION THREAD

**Discussion by wa684572** [KonataDev/Konata.Core#198] [General]
各位，有一些账号过了滑块验证，就提示Status => HighRiskEnvironment
Disconnect, Reason => Wtlogin failed.
这个怎么弄

**Comment by TheSnowfield** [comment]
更換其他的協議登錄試試看, 目前來説android pad協議風控是比較鬆的。

**Reply by AkatsukiNoKag** [reply]
///Wtlogin denied from server
Status => LoginDenied
Disconnect, Reason => Wtlogin failed.
您好，我的返回这个，能说一下具体原因吗

**Reply by TheSnowfield** [reply]
使用develop的最新commit重新登錄一次就能看到具體原因了.

**Reply by wa684572** [reply]
> 使用develop的最新commit重新登錄一次就能看到具體原因了.
最近腾讯是不是加强风控了，所有号都登录不上，android pad怎么切换
